src/components/Visit/visitlist.tsx

```diff
@@ -42,14 +41,15 @@ export const VisitList = (props: VisitListProps) => {
           })
           .map((post, index) => (
```

Comment:
The list should be sorted by most recent
we can create an issue to reverse the order